### PR TITLE
Updating documentation for testing core tool in local

### PR DESCRIPTION
**Pull request by ASHIQUEMD**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,31 +73,53 @@ Dotnet pre-compiled templates are currently hosted by the following clients. Ple
...
+    > To test dotnet-inproc, copy `Microsoft.Azure.WebJobs.ItemTemplates.X.X.X` and `Microsoft.Azure.WebJobs.ProjectTemplates.X.X.X`.
+  
+    > To test dotnet-isolated, copy `Microsoft.Azure.Functions.Worker.ItemTemplates.X.0.0` and `Microsoft.Azure.Functions.Worker.ProjectTemplates.X.0.0`.
 7. Open the templates cache directory for release version matching the one found in step 5: `%userprofile%\AppData\Local\AzureFunctionsTools\Releases\<releaseVersion>` 
 8. Open the `templates` folder for the framework you want to test:
-    1. For in-proc, use the `templates` folder fould at the root of the templates cache directory
+    1. For in-proc, use the `templates` folder found at the root of the templates cache directory
     2. For net7-isolated, use the `net7-isolated/templates` folder (for isolated, you should see a folder for netfx, net6, net5 etc.)
-9. Replace the contents of the folder with the renamed package found in `..\bin\VS`
+9.  Replace the contents of the folder with the copied packages found in `..\bin\VS`
```
Comment (liliankasem) [inline README.md:83]:
```suggestion
9. Replace the contents of the folder with the copied packages found in `..\bin\VS`

```

Comment (liliankasem) [reply README.md:83]: So we just need to copy the template packages over and don't need to rename them? That's good to know!

**Review by liliankasem — APPROVED** [review]
Thanks for the doc updates! Would be great to get sign off from @soninaren as well as he is most familiar with the process

**Comment by soninaren** [comment]
@ASHIQUEMD Thank you for the PR on doc. I really appreciate taking the extra time and effort to update the instructions. While changes to some of the sections look right. Some of your instructions call for building and publishing core tools. This may not be necessary. Let's sync on it to better understand what may be going on.

**Comment by liliankasem** [comment]
@soninaren and @ASHIQUEMD - did you sync on this PR yet?

**Comment by soninaren** [comment]
Closing this one for now. Will connect with @ASHIQUEMD and open a new PR if necessary.

**Pull request closed without merging**